scripts/generate_embeddings.py

```diff
@@ -27,8 +46,13 @@ def file_metadata_func(file_path: str) -> Dict:
         + file_path.removeprefix(EMBEDDINGS_ROOT_DIR).removesuffix("txt")
         + "html"
     )
-    print(docs_url)
-    return {"docs_url": docs_url}
+    msg = docs_url
+    if not ping_url(docs_url):
+        sys.exit(f"invalid docs_url {docs_url}")
```

Comment:
as a follow up i'd consider accumulating all the documents w/ invalid urls and then printing all of them at the end w/ the failure.  Also more information about the document that produced the invalid url would be helpful for someone who hits this in the future.

but for now it's good enough as it at least ensures we won't publish an index w/ broken urls.

Reply:
addressed in https://github.com/openshift/lightspeed-rag-content/pull/10